src/curated-corpus/helpers/topics.test.ts

```diff
@@ -89,12 +89,12 @@ describe('helper functions related to topics', () => {
 
     it('returns a list of topics without "Coronavirus" if requested', () => {
       const topics = getGroupedTopicData(data, true, false);
-      expect(topics).to.be.an('array').with.lengthOf(15);
+      expect(topics).to.be.an('array').with.lengthOf(16);
     });
 
     it('returns a full list of topics if requested', () => {
       const topics = getGroupedTopicData(data);
-      expect(topics).to.be.an('array').with.lengthOf(16);
+      expect(topics).to.be.an('array').with.lengthOf(17);
```

Comment:
With most of the test updates involving updating the number of topics expected, should tests be modified to not check for the total number of topics? Should we compare them to the enums in the generated code instead? 

Or is adding a topic going to be a rare enough event that it's fine to leave this - and by extension the process of adding a topic - as-is?

Reply:
good question and i'm not sure we know the answer yet re: how often will we get new topics. but - i think your point about avoiding testing a specific number of topics is sound. we should stop testing the number - i think testing against the enum sounds good.